src/sentry/sentry_metrics/db.py

```diff
@@ -0,0 +1,2 @@
+POSTGRES_DB = "postgres"
+CLOUDSPANNER_DB = "cloudspanner"
```

Comment:
if we wanted we could split cloudspanner into like:

```python
CLOUDSPANNER_SINGLE_DB = "cloudspanner-single"
CLOUDSPANNER_MULTI_DB = "cloudspanner-multi"
```

for single and multi cloudspanner instances, assuming that you'd have the `use_case_id` in the configuration be the indicator for the db within the instance

Reply:
I would not add this db.py file with these strings. I think you'd be better off with an enum in the configuration.py file like how the rest of the options for running the indexer are defined.

Reply:
okay that works, wasn't too sure about putting these here anyway